### DISCUSSION TITLE
Plugin that will show coordinates.

### DISCUSSION THREAD

**Discussion by comdar16** [Pryaxis/TShock#2482] [Show and tell]
https://github.com/COMDARR/GetPos

Tshock has /tppos command but it doesn't have /getpos command so i decided to make my own.

Source code and compiled .dll are provided!

**Comment by comdar16** [comment]
Basicly /tppos is useless if you can't get any info about your own coordinates

**Comment by bartico6** [comment]
/pos

https://github.com/Pryaxis/TShock/blob/general-devel/TShockAPI/Commands.cs#L454

**Reply by comdar16** [reply]
> /pos
> 
> https://github.com/Pryaxis/TShock/blob/general-devel/TShockAPI/Commands.cs#L454

The command existed but looks like i never seen it before.
